packages/odyssey-react-mui/src/OdysseyProvider.tsx

```diff
@@ -33,11 +35,15 @@ const OdysseyProvider = ({
   children,
   nonce,
   themeOverride,
+  tokensOverride,
```

Comment:
**NON-BLOCKING**Do you think we could rename this `designTokensOverride`? I think that makes more sense than `tokensOverride` because it describes the tokens, but it's up to you.

Reply:
I'll get this in a separate PR